rust-connectors/sources/mqtt/src/main.rs

```diff
@@ -74,6 +44,8 @@ fn main() -> Result<(), MqttConnectorError> {
         "Starting MQTT source connector",
     );
 
+    debug!("opts={:?}", opts);
```

Comment:
can you change it tracing log format `debug!(opts)`

Reply:
it complains about `fluvio_future::tracing::Value` is not satisfied

Reply:
debug!(opt = ?opt)

Reply:
Done